src/main/java/org/broadinstitute/hellbender/utils/tsv/TableReader.java

```diff
@@ -365,8 +365,7 @@ private R fetchNextRecord() throws IOException {
     }
 
     private void processCommentLine(final String[] line, final long lineNumber) {
-        final StringBuilder builder = new StringBuilder();
-        builder.append(line[0].substring(TableUtils.COMMENT_PREFIX.length()));
+        final StringBuilder builder = new StringBuilder(line[0]);
```

Comment:
Did you mean `final StringBuilder builder = new StringBuilder(line[0].substring(TableUtils.COMMENT_PREFIX.length()));`?

Reply:
This is causing tests to fail.

Reply:
Fixed the bug... The comment starting character (in fact the first character of the comment line) was stripped so I was loosing the "@". This only affects another place and went it and changed it...